front/gatsby/src/util/helpers.ts

```diff
@@ -23,16 +24,14 @@ export function stripAddressFromCartItem(item: string): string {
   return item.slice(5);
 }
 
-export function getCartItemsCount(): number {
-  let count = 0;
+export function addToCart(stash: string, currCartCount: number): void {
+  writeStorage(`cart:${stash}`, stash);
```

Comment:
Why not have one key in localStorage called `cart`, which is an array of stash addresses.
```
const [cart] = useLocalStorage('cart');
const cartCount = cart.length; // maybe you would need to JSON.parse(cart), i'm not sure
```

Reply:
yeah, it is redundant, but I think read + parse would be more laggy than simply read + it's only a single key extra.